lib/openid_connect/authorization_endpoint.rb

```diff
@@ -0,0 +1,10 @@
+class AuthorizationEndpoint
+  attr_accessor :app, :account, :client, :redirect_uri, :response_type, :scopes, :_request_, :request_uri, :request_object
+  delegate :call, to: :app
+
+  def initialize(allow_approval = false, approved = false)
```

Comment:
Unused method argument - `allow_approval`. If it's necessary, use `_` or `_allow_approval` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `approved`. If it's necessary, use `_` or `_approved` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space detected in default value assignment.